doc/onboarding-extras.md

```diff
@@ -2,23 +2,29 @@
 
 | subsystem | maintainers |
 | --- | --- |
-| `lib/buffer` | @trevnorris |
-| `lib/child_process` | @cjihrig, @bnoordhuis |
-| `lib/cluster` | @cjihrig, @bnoordhuis |
+| `lib/assert` | @Trott |
+| `lib/buffer` | @nodejs/buffer |
+| `lib/child_process` | @bnoordhuis, @cjihrig |
+| `lib/cluster` | @bnoordhuis, @cjihrig, @mcollina |
 | `lib/{crypto,tls,https}` | @nodejs/crypto |
 | `lib/domains` | @misterdjules |
-| `lib/{_}http{*}` | @indutny, @bnoordhuis, @mscdex, @nodejs/http |
-| `lib/net` | @indutny, @bnoordhuis, @nodejs/streams |
+| `lib/fs`, `src/{fs|file}` | @bnoordhuis, @jasnell, @saghul, @trevnorris |
+| `lib/{_}http{*}` | @nodejs/http |
+| `lib/net` | @bnoordhuis, @indutny, @nodejs/streams |
 | `lib/{_}stream{s|*}` | @nodejs/streams |
 | `lib/repl` | @fishrock123 |
 | `lib/timers` | @fishrock123, @misterdjules |
-| `lib/zlib` | @indutny, @bnoordhuis |
+| `lib/util` | @bnoordhuis, @cjihrig, @evanlucas |
+| `lib/zlib` | @addaleax, @bnoordhuis, @indutny |
 | `src/async-wrap.*` | @trevnorris |
 | `src/node_crypto.*` | @nodejs/crypto |
 | `test/*` | @nodejs/testing |
 | `tools/eslint`, `.eslintrc` | @silverwind, @trott |
-| upgrading v8 | @bnoordhuis, @targos, @ofrobots |
-| upgrading npm | @thealphanerd, @fishrock123 |
+| upgrading v8 | @nodejs/v8, @nodejs/post-mortem |
+| upgrading npm | @fishrock123, @thealphanerd |
```

Comment:
I disagree that we should be added to that. That team is for notifying npm. Usually we are the ones *merging* npm prs. It's a bit weird.